Tests/ModelHelpers/AttributesTest.php

```diff
@@ -134,6 +128,20 @@ public function testGetAttributeValue()
 		);
 	}
 
+	/**
+	 * @return void
+	 */
+	public function testAttributeNotFoundException()
```

Comment:
Still into different functions. One for `getAttributeValue` and one for `getAttribute`